CMakeLists.txt

```diff
@@ -10,7 +10,7 @@ set(LIBFM_QT_LIBRARY_NAME "fm-qt" CACHE STRING "fm-qt")
 
 set(LIBFM_QT_API_VERSION_MAJOR 1)
 set(LIBFM_QT_API_VERSION_MINOR 0)
-set(LIBFM_QT_API_VERSION_PATCH 0)
+set(LIBFM_QT_API_VERSION_PATCH 1)
```

Comment:
Please don't change this. Because of the changes that are made to libfm-qt and pcmanfm-qt, we can't make a point release. The versions will be set when LXQt 1.1.0 is released.

Reply:
Are there ABI incompatible changes after 1.0.0 release?

Reply:
OK. Then we can't do the proper release of xdg-desktop-portal-lxqt :(

Reply:
What about the initial release now?